### PR TITLE
Blog Post: The StackHawk ZAP Fund

### PR DESCRIPTION
**Pull request by psiinon**
Screenshot 2022-03-22 at 15-15-04 The StackHawk ZAP Fund

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+![StackHawk and ZAPBot](images/hawk_bot.png)
+
+Obviously the ZAP Core Team is very grateful for this initiative and have been working with [StackHawk](https://www.stackhawk.com) to make the most of this great opportunity.
+
+As you will see from the main [ZAP Fund](https://www.stackhawk.com/zap-fund/) page there are a set of ZAP issues which now have bounties on them, including some “rolling” issues which we will keep adding until the relevant features have all been implemented.
+
+## Getting Started
+
+In order to work on any of these issues you will need to set up a ZAP development environment.
```
Comment (kingthorin) [inline site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md:26]: Ummm should set up be all one word here? "setup"

File: site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Then you just need to choose an issue to work on - these are listed on the [ZAP Fund](https://www.stackhawk.com/zap-fund/) page and are tagged with the [stackhawk-bounty](https://github.com/zaproxy/zaproxy/issues?q=is%3Aissue+is%3Aopen+label%3Astackhawk-bounty) label in our main repo.
+
+Just comment on an unassigned issue to reserve it. In order to give everyone a fair chance only one [stackhawk-bounty](https://github.com/zaproxy/zaproxy/issues?q=is%3Aissue+is%3Aopen+label%3Astackhawk-bounty) issue will be assigned to any one person at a time.
+
+## Scan Rules and Spiders
+
+Many of the initial set of issues are related to the scan rules and the spiders. This is partly because these are relatively self contained, and so should be easier to get to grips with.
+
+They are also easier to test and are covered by some test applications which we are now testing with ZAP every day - the results are on https://www.zaproxy.org/docs/scans/
```
Comment (kingthorin) [inline site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md:37]: Period?

File: site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  StackHawk has launched a $100,000 ZAP Fund dedicated to improving ZAP and the ZAP Community.
+images:
+- https://www.zaproxy.org/blog/2022-03-22-the-stackhawk-zap-fund/images/hawk_bot.png
+type: post
+tags:
+- blog
+date: "2022-03-22"
+authors: 
+    - simon
+---
```
[the remark this answers is not in the document]
Comment (psiinon) [reply site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md:13]: Not any more - thats cascaded from one level up :)

File: site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md (hunk 1 above)
Comment (psiinon) [reply site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md:26]: It looks like it should be 2 words, according to https://grammarist.com/spelling/set-up-vs-setup/ :D

**Comment by psiinon** [comment]
Just made the zaproxy.org links relative - thanks @thc202  ;)

**Review by ricekot — APPROVED** [review]

File: site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md (hunk 1 above)
Comment (kingthorin) [reply site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md:26]: Fair, I was waffling about it that's why I asked 😀

**Pull request merged**